docs/source/install/deb.rst

```diff
@@ -30,13 +30,12 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
 Setup repositories
 ~~~~~~~~~~~~~~~~~~~
 
-  .. code-block:: bash
+The following script will detect your platform and architecture and setup the repo accordingly. It'll also install the GPG key for repo signing.
```

Comment:
From Bintray stats - we got 2-5 downloads/daily for new packages.
I don't think it makes sense to provide migration script, considering big red `BETA` notice: everything is OK to change with it.

---

Apart of that, I remember you wanted to provide redirect links.
Feel free to commit directly when you guys decided something.